cmd/terramate/cli/cloud.go

```diff
@@ -347,14 +345,15 @@ func (c *cli) doPreviewAfter(run stackCloudRun, res runResult) {
 	}
 
 	if !c.parsedArgs.Quiet {
```

Comment:
the remove this `if` as well because the `--quiet` only controls the printing messages, not the logging.